### PR TITLE
update enodes for ropsten and main clusters

### PR DESCRIPTION
**Pull request by gravityblast**
This PR updates `BootNodes` and `StatisNodes` for `ropstenCluster` and `mainnetCluster`.

It could be refactored since we are using the same `enodes` in both, but I think it's better to wait to see if we need other changes for beta.

### CODE REVIEW

File: geth/params/cluster.go [hunk 1]
```diff
@@ -9,12 +9,22 @@ type cluster struct {
...
 	BootNodes: []string{
-		"enode://f32efef2739e5135a0f9a80600b321ba4d13393a5f1d3f5f593df85919262f06c70bfa66d38507b9d79a91021f5e200ec20150592e72934c66248e87014c4317@167.99.209.79:30404",
-		"enode://90d7e7e34f588b44e1e07adfc6453b0d27946883420a012a837f6acfa66cbb28e6e8b5df3c0e26615ed82b32273453b1b003e4aa5c0d357ada30bde8f666db47@167.99.46.141:30404",
+		"enode://8472a9a236afe091e9941909a23b8e987287ad76839da2d9260b3d1697fea180d494617d42e605a5c229c7bf326e2bfa3d1a390fd03bf1783376c275e380126e@206.189.108.52:30404",
+		"enode://c266882060e6670030fac53f07dfd802d51441c72b188722352d964095ab79f6aa1350b9000a2ab3a2d76a835c79ffcf6cd022a656ec04e7a02683417364ce5e@128.199.55.181:30404",
 	},
 	StaticNodes: []string{
-		"enode://dffef3874011709b12d1e540d83ddb19a9db8614ad9151d05bcf813585e45cbebba5aaea223fe315786c401d8cecb1ad2de9f179680c536ea30311fb21fa934b@188.166.100.178:30303",
-		"enode://03f3661686d30509d621dbe5ee2e3082923f25e94fd41a2dd8dd34bb12a0c4e8fbde52247c6c55e86dc209a8e7c4a5ae56058c65f7b01734d3ab73818b44e2a3@188.166.33.47:30303",
+		"enode://a6a2a9b3a7cbb0a15da74301537ebba549c990e3325ae78e1272a19a3ace150d03c184b8ac86cc33f1f2f63691e467d49308f02d613277754c4dccd6773b95e8@206.189.108.68:30304",
```
Comment (dshulyak) [inline geth/params/cluster.go:16]: the idea was to add 2 any static node, not all of them

**Comment by gravityblast** [comment]
sorry my fault, I'll update it leaving just two of them

**Review by dshulyak — APPROVED** [review]

**Review by adambabik — APPROVED** [review]

**Comment by adambabik** [comment]
@jakubgs do we have any means to flag the Whisper nodes that we selected here to be fallback nodes in Consul? That may be a valuable information in the cluster.

**Comment by jakubgs** [comment]
@adambabik sure we do, they are registered in Consul as services:
http://consul.status.im/v1/catalog/service/statusd-whisper
We could use the [resgister-entity](https://www.consul.io/api/catalog.html#register-entity) call for the Catalog to update the service tags(add `backup` tag) or Meta data to add something like `backup: true`.

<blockquote><div><strong><a href="http://consul.status.im/v1/catalog/service/statusd-whisper">Sign In</a></strong></div></blockquote>

**Review by jakubgs — APPROVED** [review]
Looks good to me, but I'd like to know if there is any simple way of testing the enode address to make sure it receives connections and speakes the protocol.

**Review by jakubgs — CHANGES_REQUESTED** [review]
Both of the static whisper nodes you've chosen are on the same host: `206.189.108.62`
You should pick another one.

**Comment by gravityblast** [comment]
@jakubgs I updated it using 4 different hosts, 2 for ropsten and two for mainnet

**Review by jakubgs — APPROVED** [review]
LGTM!

**Comment by adambabik** [comment]
> We could use the resgister-entity call for the Catalog to update the service tags(add backup tag) or Meta data to add something like backup: true.

Let's do it after Status release.

**Pull request merged**